src/serviceFabricDiscoveryService/pkg/discovery/labels.go

```diff
@@ -64,7 +64,7 @@ func setLoadbalancerSticky(lb *dynamic.ServersLoadBalancer, val string) error {
 
 func setLoadbalancerHealthcheckPath(lb *dynamic.ServersLoadBalancer, val string) error {
 	if lb.HealthCheck == nil {
-		lb.HealthCheck = &dynamic.HealthCheck{}
+		lb.HealthCheck = &dynamic.ServerHealthCheck{}
```

Comment:
Did the labels change between traefik versions? can you explain the reason in modifying this? if it is because of an updated traefik version what version was this introduced in? Looking at this https://doc.traefik.io/traefik/routing/services/#health-check

Reply:
@egaribay77 labels haven't change in between version in v2.
I have done this change because I have updated `github.com/traefik/genconf` it is a package that allow you to have a representation of the dynamic configuration without depending of `github.com/traefik/traefik`

Since Traefik v2.5 the healthcheck doesn't exist anymore. It has been replaced by `ServerHealthCheck` that has been introduce by https://github.com/traefik/traefik/pull/8057 as a non breaking change in Traefik.